content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,186 @@
+  ([Issue #25678](https://github.com/istio/istio/issues/25678))
+
+- **Fixed** when a node has multiple IP addresses (e.g., a VM in the mesh expansion scenario),
+Istio Proxy will now bind `inbound` listeners to the first applicable address in the list
+(new behavior) rather than to the last one (former behavior).
+  ([Issue #28269](https://github.com/istio/istio/issues/28269))
+
+## Security
+
+- **Improved** Gateway certificates to be read and distributed from Istiod, rather than in the gateway pods.
+This reduces the permissions required in the gateways, improves performance, and will make certificate reading
+more flexible in the future. This change is fully backwards compatible with the old method, and requires no changes
+to your cluster. If required, it can be disabled by setting the `ISTIOD_ENABLE_SDS_SERVER=false`
+environment variable in Istiod.
+
+- **Improved** TLS configuration on sidecar server side inbound paths to enforce `TLSv2` version along with recommended cipher suites.
+             If this is not needed or creates problems with non Envoy clients, it can disabled by setting env variable `PILOT_SIDECAR_ENABLE_INBOUND_TLS_V2` to false.
+
+- **Updated** The `ipBlocks`/`notIpBlocks` fields of an `AuthorizationPolicy` now strictly refer to the source IP address of the IP packet as it arrives at the sidecar.  Prior to this release, if using the Proxy Protocol, then the `ipBlocks`/`notIpBlocks` would refer to the IP address determined by the Proxy Protocol.  Now the `remoteIpBlocks`/`notRemoteIpBlocks` fields must be used to refer to the client IP address from the Proxy Protocol.
+ ([reference](/docs/reference/config/security/authorization-policy/))([usage](/docs/ops/configuration/traffic-management/network-topologies/))([usage](/docs/tasks/security/authorization/authz-ingress/)) ([Issue #22341](https://github.com/istio/istio/issues/22341))
+
+- **Added** `AuthorizationPolicy` now supports nested JWT claims.
+  ([Issue #21340](https://github.com/istio/istio/issues/21340))
+
+- **Added** support for client side Envoy secure naming config when trust domain alias is used.
+This fixes the multi-cluster service discovery client SAN generation to use all endpoints' service accounts rather than the first found service registry.
+
+- **Added** Experimental feature support allowing Istiod to integrate with external certificate authorities using Kubernetes CSR API (>=1.18 only).
+  ([Issue #27606](https://github.com/istio/istio/issues/27606))
+
+- **Added** Enable user to set the custom VM identity provider for credential authentication
+  ([Issue #27947](https://github.com/istio/istio/issues/27947))
+
+- **Added** action 'AUDIT' to Authorization Policy that can be used to determine which requests should be audited.
+  ([Issue #25591](https://github.com/istio/istio/issues/25591))
+
+- **Added** support for migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
```

Comment:
@myidpt do you know who worked on this and can we add a link to the github issue? thanks

Reply:
no immediate issue found, will link the original PR: https://github.com/istio/istio/pull/26482